docker-compose.yml

```diff
@@ -6,6 +6,9 @@ services:
     ports:
       - "30333:5000"
 
+    tmpfs: 
+      - /tmp
```

Comment:
I would suggest adding a size limit to tmpfs since you could use up all memory on the host pretty easily. 

Perhaps something like this: 
```
    tmpfs:
      - /tmp:size=5G
```
Details here https://docs.docker.com/storage/tmpfs/